Resources/public/css/styles.css

```diff
@@ -345,6 +345,28 @@ td.sonata-ba-list-field .editable-empty:not(.editable-open) {
     width: auto; /* See https://github.com/sonata-project/SonataAdminBundle/issues/2950 */
 }
 
+/* icheck */
+.checkbox div[class^="icheckbox"],
```

Comment:
Sorry but I still don't understand why you're putting Icheck CSS here.

iCheck is part of the AdminLTE theme on vendor section.